source/markdown/steekproefontwerp_zandstreek.Rmd

```diff
@@ -889,7 +889,9 @@ allocatie <- tar_read(allocatie_df, store = targets_store) %>%
   ))
 
 allocatie %>%
-  mutate(steekproeftrekking = round(popsize / 20)) %>%
+  mutate(steekproeftrekking = ifelse(popsize < 2000,
+                                     round(popsize / 5),
+                                     round(popsize / 20))) %>%
   select(-c(openheid_sbp, targetsize, excess)) %>%
```

Comment:
Deze wijziging is nog niet gereflecteerd in de begeleidende tekst. Kan je dat aanpassen?